src/SFA.DAS.Assessor.Functions/Functions/Print/PrintFunctionFlow.cs

```diff
@@ -22,6 +22,11 @@ public async Task Run([TimerTrigger("%FunctionsSettings:CertificatePrintFunction
             {
                 if (myTimer.IsPastDue)
                 {
+                    if(myTimer.ScheduleStatus.Last < DateTime.UtcNow.AddMinutes(-1))
+                    {
+                        log.LogCritical("Epao Importer PrintFunctionFlow timer trigger is running more than 1 minute late");
```

Comment:
do we want to return here and not trigger the function

Reply:
That bit of code was added by mistake I took it out above in commit a806a54

Reply:
so you did, don't know how I missed that